inc/template-tags/display-svg.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Display SVG Markup.
+ *
+ * @package _s
+ */
+
+namespace WD_S\TemplateTags;
```

Comment:
I also would prefer using `WebDevStudios\` in the namespace:

```
namespace webdevstudios\wd_s\*\*
```

And I would expect all our projects to follow this format.

Reply:
@salcode I know we somewhat shared opinions on this? What do you think?

Reply:
My primary thought is I don't want @batesweb to have to keep modifying this PR 😆 , perhaps we want to have a small group call to come to a consensus on direction.

Personally:
1. I agree with preferring the full company name for the namespace. I think it is good for our brand and more inline with other namespaces I see being used.
2.  I'd prefer PascalCase, `WebDevStudios`, rather than all lower-case `webdevstudios`). I like PSR-4ish namespaces.
3. Using sub-namespaces `WebDevStudios\functions` and `WebDevStudios\TemplateTags` is an interesting idea that I'm open to but not locked in on in terms of an opinion
4. For function name prefixes, I'd much prefer `get_` prefixed functions rather than `return_`

Related namespace reading https://make.wordpress.org/core/2020/03/20/updating-the-coding-standards-for-modern-php/ (TLDR: This advocates for Camel_Caps for namespaces, e.g. `My_Awesome_Plugin\Admin`, which I think would still be `WebDevStudios` since it is one word).

Reply:
I would totally be OK with `PascalCase` as long as we use `_`'s e.g. `\WebDevStudios\Functions\Pascal_Case`, etc :D

Reply:
After giving this more thought, I think we should use the same namespace, `WebDevStudios\wd_s`, on all files.

i.e.

```
namespace WebDevStudios\wd_s;
```

1. `{organization}\{product}` is a common format for namespaces, in this case our product is `wd_s`
2. I like the all lower-case `wd_s` because, based on the [README.md](https://github.com/WebDevStudios/wd_s#readme), the name of this product _is all lowercase_
3. Using one namespace for the entire project
    - Advantage: we don't have to worry about what namespace a function is in, they are all in the same one
    - Advantage: we don't need any `use function` lines because we don't need to import a function from another namespace
    - Disadvantage: we can't re-use the same function name in two different places (e.g. in `functions` and in `TemplateTags`) - I don't think we should be doing this anyway
    - Advantage: If we want to change the namespace (e.g. a client doesn't want our branding in the namespace) it is a relatively simple search and replace across all of the files
    - Note: when we do need to use the namespace (e.g. in a `add_action()` or `add_filter()`, see examples in blog post [WordPress Hooks and PHP Namespaces](https://salferrarello.com/wordpress-hooks-and-php-namespaces/)), we should always use `__NAMESPACE__` to maintain the ability to change the namespace

Reply:
> I think we should use the same namespace, `WebDevStudios\wd_s`, on all files [...] Disadvantage: we can't re-use the same function name in two different places

I think expanding on the namespacing ensures comfort and ease knowing you will have less function name conflicts, e.g. I can have a `get_my_slug()` in two different context/files vs having to prefix it. Hence why I would disagree here. 

But not strongly, but I think being able to expand on Namespaces should be allowed and encouraged. I think either way we go is going to be _better_! That being said, I personally like the direction @batesweb is going here, and I would keep it e.g. `WebDevStudios\wd_s\Template_Tags`, `WebDevStudios\wd_s\Functions`, etc.

Reply:
I brought this topic to the larger BEE group at the 2022-01-06 BEE Scrum and the general consensus was to use a single namespace across all files of

```
namespace WebDevStudios\wd_s;
```

Reply:
To add context:

* There are currently no classes in `wd_s` the namespace is not used for directory/path autoloading
* The purpose of adding namespaces was to eliminate global function prefixing ala `function _s_something()`
* By having one namespace, we'll never have to guess which namespace we're in and all hooks can simply use the `__NAMESPACE__` constant

Reply:
The BEE team discussed `return_` && `display_` (vs. WP's `get_` && `the_`). We agreed that WP is just wrong with `the_` because it doesn't describe what it's doing.

However, we unanimously agreed that `get_` should be kept instead of `return_` because it just makes sense and with the namespace added, there should be no confusion where the function lives.

Reply:
BEEs like `print_` better than `display_`